### PR TITLE
Changes exported constants naming convention.

### LINKED ISSUE #24: Use consistent naming scheme for constants
I did a quick search in `skynet-js` for `export const`
```
src/download.ts:
  79: export const defaultDownloadOptions = {
  80    ...defaultOptions("/"),

src/registry.ts:
  26: export const defaultGetEntryOptions = {
  27    ...defaultOptions("/skynet/registry"),
  35: export const MAX_GET_ENTRY_TIMEOUT = 300; // 5 minutes
  40: export const regexRevisionNoQuotes = /"revision":\s*([0-9]+)/;

src/mysky/index.ts:
  26: export const mySkyDomain = "skynet-mysky.hns";
  27  const mySkyUiRelativeUrl = "ui.html";

src/utils/number.ts:
  4: export const MAX_REVISION = BigInt("18446744073709551615"); // max uint64

src/utils/skylink.ts:
  33: export const uriHandshakePrefix = "hns:";
  34: export const uriHandshakeResolverPrefix = "hnsres:";
  35: export const uriSkynetPrefix = "sia:";

src/utils/url.ts:
  10: export const defaultSkynetPortalUrl = "https://siasky.net";
  ```
 
doesn't really seem like we're consistently using a schema at the moment. Changing it doesn't even have to break compatibility either, we could just do something like
  
  ```
  // @deprecated please use URI_HANDSHAKE_PREFIX
  export const uriHandshakePrefix = "hns:"

  export const URI_HANDSHAKE_PREFIX = "hns:"
  ```
  and then just adopt `URI_HANDSHAKE_PREFIX` everywhere.

Credit: @peterjan

### PR DESCRIPTION
**Pull request by amberm31**
Closes #24.

### CODE REVIEW

**Comment by m-cat** [comment]
The failing checks are not due to this Pull Request and will be fixed by https://github.com/SkynetLabs/skynet-js/pull/235.

File: src/registry.ts [hunk 1]
```diff
@@ -42,29 +42,53 @@ export type CustomSetEntryOptions = BaseCustomOptions & {
...
+ * Regex for JSON revision value without quotes.
+ *
+ * @deprecated please use REGEX_REVISION_NO_QUOTES.
+ */
+export const regexRevisionNoQuotes = REGEX_REVISION_NO_QUOTES;
+
+/**
+ * Regex for JSON revision value with quotes.
+ */
+export const REGEX_REVISION_WITH_QUOTES = /"revision":\s*"([0-9]+)"/;
```
Comment (m-cat) [inline src/registry.ts:84]: This shouldn't be exported if the original const was not exported. Same with `regexRevisionWithQuotes` below.

```suggestion
const REGEX_REVISION_WITH_QUOTES = /"revision":\s*"([0-9]+)"/;
```

File: src/registry.ts [hunk 2]
```diff
@@ -42,29 +42,53 @@ export type CustomSetEntryOptions = BaseCustomOptions & {
...
 
 /**
  * Regex for JSON revision value with quotes.
+ *
+ * @deprecated please use REGEX_REVISION_WITH_QUOTES.
  */
-const regexRevisionWithQuotes = /"revision":\s*"([0-9]+)"/;
+export const regexRevisionWithQuotes = REGEX_REVISION_WITH_QUOTES;
```
Comment (m-cat) [inline src/registry.ts:91]:
```suggestion
const regexRevisionWithQuotes = REGEX_REVISION_WITH_QUOTES;
```

**Review by m-cat — CHANGES_REQUESTED** [review]
Looks fantastic! Just a couple minor changes needed. Thank you for your contribution, and welcome to GitHub!

**Review by m-cat — APPROVED** [review]
Great job!

**Review by kwypchlo — APPROVED** [review]

**Review by peterjan — APPROVED** [review]

**Comment by peterjan** [comment]
I like that we're introducing this naming convention! 

Something that might be useful to introduce in the future is a `consts.deprecated.ts` file where we gather all deprecated constants. The obvious benefit is that we don't have all these old constants in our "live" codebase, and we get rid of those nasty deprecation warnings.

**Pull request merged**